hw-2/server.ts

```diff
@@ -0,0 +1,64 @@
+import express = require('express');
+import {UsersStorage} from './storage/users';
```

Comment:
Surround object properties and array elements with spaces - please add this rule to lint